manifests/init.pp

```diff
@@ -29,6 +31,8 @@
   $slaveof            = undef,
   $version            = 'installed',
   $packages           = ['redis']
+  $redis_conf         = 'redis.conf',
+  $redis_service      = 'redis.service',
```

Comment:
I'd recommend setting this to just _redis_ so that its compatible with non-systemd still.